json-schemas/src/client-events-connections.json

```diff
@@ -0,0 +1,65 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Ably Connections Client Event",
+  "type": "object",
+  "description": "A message sent on the clientEvents:connections metachannel describing a sampled client connection.",
+  "properties": {
+    "host": {
+      "type": "string",
+      "description": "The HTTP Host header sent with the connection."
```

Comment:
You don't really _send_ a connection. How about:

"The HTTP Host header used when establishing the connection."

(and similarly in other descriptions, replace "making" or "sending" with "establishing" a connection)

Reply:
I thought it was implied that this meant "sent with the connection request", but I'm ok with your suggestion.